core-bundle/src/Image/Studio/FigureBuilder.php

```diff
@@ -370,8 +371,14 @@ public function setLinkAttribute(string $attribute, ?string $value, bool $forceR
      * remove an auto-generated value from the results, set the respective
      * attribute to null.
      */
-    public function setLinkAttributes(array $attributes): self
+    public function setLinkAttributes(array|HtmlAttributes $attributes): self
```

Comment:
You sure you don't want to break here too?

Reply:
Or have it the other way around? `HtmlAttributes::createFromArray()` or so?

Reply:
I’d vote for not breaking here, so still allowing the `array` type too.

Reply:
But could convert to `HtmlAttributes` internally ;)

Reply:
We need to keep this internal representation, because it also allows to mark which values of the default set will be removed after building. The `HtmlAttributes` class does not store falsy values.